lib/aptly/snapshot.rb

```diff
@@ -50,6 +50,23 @@ def publish(prefix, **kwords)
       Aptly.publish([{ Name: self.Name }], prefix, 'snapshot', kwords)
     end
 
+    # Lists all PublishedRepositories self is published in. Namely self must
```

Comment:
Makes sense, I was thinking of putting it in Representation at first, but that seemed like bad design. I'll add a new Module.